assets/app/views/project.html

```diff
@@ -75,7 +75,8 @@ <h2 class="service" ng-if="!displayRouteByService[serviceId]">
                 <span ng-if="service.spec.portalIP != 'None'">
                   routing traffic on {{service.spec.portalIP}}
                 </span>
-                <span ng-if="numPorts">&#8211;
+                <span ng-if="numPorts">
```

Comment:
Sorry, I mean change this `span` element to a `div`. Then you don't need the empty `div` below. (Avoid block elements inside inline elements.)